app/views/application/_footer.html.erb

```diff
@@ -0,0 +1,5 @@
+<footer role="contentinfo" class="footer">
+  <div class="container">
+    Learn more about <a href="http://projectcypress.org">Project Cypress</a>, an open source project sponsored by CMS and ONC.<br> Need help? <a href="http://projectcypress.org/documents/cypress_user_guide_prs_12_5071.pdf">Click here</a>.
```

Comment:
I'd adjust this link from the ambiguous "Click Here" to clarify that you're about to click on the Cypress User Guide!